portfolio/src/main/java/com/google/sps/servlets/BlogSentimentServlet.java

```diff
@@ -27,7 +28,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
                 newBlog = newBlog + s;
                 continue;
             }
-            newBlog = newBlog + "<br><br>" + s;
+            if(s.charAt(s.length()-1)=='>'){
```

Comment:
A comment before this if statement explaining the situation you're catching would be helpful for other readers (I think this is basically looking for the end of a paragraph? not sure though)

Reply:
Yep, it's looking for things like </a> and </p>. Not sure if there's a better solution but visually this produces the right result for now.